src/main/webapp/components/lists/lists.js

```diff
@@ -97,20 +125,41 @@ const Lists = props => {
           </ListTitle>
         )
 
-        return (
+        return list['list_bookmarks'] == undefined ? (
```

Comment:
:pencil2:
```suggestion
        return list['list_bookmarks'] === undefined ? (
```

Reply:
I've been having alot of issues with list['list_bookmarks'] being null as opposed to undefined.